OpenOversight/migrations/versions/2023-08-21-0300_a35aa1a114fa_add_create_and_last_update_columns.py

```diff
@@ -0,0 +1,289 @@
+        batch_op.drop_constraint("faces_created_by_fkey", type_="foreignkey")
+        batch_op.create_foreign_key(
+            "fk_faces_created_by_users",
+            "users",
+            ["created_by"],
+            ["id"],
+            ondelete="SET NULL",
+        )
+        batch_op.create_foreign_key(
+            "fk_faces_last_updated_by_users",
+            "users",
+            ["last_updated_by"],
+            ["id"],
+            ondelete="SET NULL",
+        )
+
+    with op.batch_alter_table("jobs", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("last_updated_at", sa.DateTime(timezone=True), nullable=True)
+        )
+        batch_op.add_column(sa.Column("last_updated_by", sa.Integer(), nullable=True))
+        batch_op.create_foreign_key(
+            "fk_jobs_last_updated_by_users",
+            "users",
+            ["last_updated_by"],
+            ["id"],
+            ondelete="SET NULL",
+        )
+
+    with op.batch_alter_table("license_plates", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("last_updated_at", sa.DateTime(timezone=True), nullable=True)
+        )
+        batch_op.add_column(sa.Column("last_updated_by", sa.Integer(), nullable=True))
+        batch_op.create_foreign_key(
+            "fk_licence_plates_last_updated_by_users",
+            "users",
+            ["last_updated_by"],
+            ["id"],
+            ondelete="SET NULL",
+        )
+
+    with op.batch_alter_table("links", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("last_updated_at", sa.DateTime(timezone=True), nullable=True)
+        )
+        batch_op.add_column(sa.Column("last_updated_by", sa.Integer(), nullable=True))
+        batch_op.create_foreign_key(
+            "fk_links_last_updated_by_users",
+            "users",
+            ["last_updated_by"],
+            ["id"],
+            ondelete="SET NULL",
+        )
+
+    with op.batch_alter_table("locations", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("last_updated_at", sa.DateTime(timezone=True), nullable=True)
+        )
+        batch_op.add_column(sa.Column("last_updated_by", sa.Integer(), nullable=True))
+        batch_op.create_foreign_key(
+            "fk_locations_last_updated_by_users",
+            "users",
+            ["last_updated_by"],
+            ["id"],
+            ondelete="SET NULL",
+        )
+
+    with op.batch_alter_table("notes", schema=None) as batch_op:
+        batch_op.alter_column("updated_at", new_column_name="last_updated_at")
+        # batch_op.add_column(sa.Column('last_updated_at', sa.DateTime(timezone=True), nullable=True))
+        batch_op.add_column(sa.Column("last_updated_by", sa.Integer(), nullable=True))
+        # batch_op.create_foreign_key(None, 'users', ['last_updated_by'], ['id'], ondelete='SET NULL')
+        # batch_op.drop_column('updated_at')
```

Comment:
Can you remove the commented alembic lines?

Reply:
Removed commented lines!